frontend/packages/knative-plugin/src/components/add/import-types.ts

```diff
@@ -2,10 +2,8 @@ import { LimitsData } from '@console/dev-console/src/components/import/import-ty
 
 export enum EventSources {
   CronJobSource = 'CronJobSource',
-  ContainerSource = 'ContainerSource',
```

Comment:
why did you removed  `ContainerSource` ?

Reply:
added it back.